content/integrations/awsdirectconnect.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+
+Enable this integration to see in Datadog all your Direct Connect metrics
+
+## Setup
+### Installation
+
+If you haven't already, set up the [Amazon Web Services integration first](/integrations/aws). There are no other installation steps that need to be performed.
+
+### Configuration
+
+In the Amazon Web Services integration tile, ensure that Direct Connect is checked under metric collection.
```

Comment:
In the Amazon Web Services integration tile in Datadog, tick the checkbox next to **Direct Connect** under **metric collection**.